blog/2023/12-11-roku-v200/index.mdx

```diff
@@ -0,0 +1,121 @@
+slug: roku-200
+tags: [release, roku]
+---
+
+When we say this 2.0.0 release is a major milestone, we honestly mean it. New features and bug fixes abound, but even behind the scenes the updates continue all the way to the foundation with us updating every single file in the app to support a new programming language. You can't get more monumental than that!
+
+Read more to learn about some of the exciting new features.
+
+\- 1hitsong
+
+{/* truncate */}
+
+Requires minimum server version of 10.8.1
+
+### 👥 User Select Screen / Auto Login
+
+![Screenshot of the user select screen in the Jellyfin Roku app](./UserSelect.gif)
+
+When launching the Jellyfin app, it now presents you with the list of all public users and any other remembered users for you to login as.
+
+If you prefer to have the app automatically login as a specific user, simply login as them and enable the new global Remember Me setting. With this global setting enabled, the app will automatically login as this user at the start of the Jellyfin app without needing to select them or enter a password.
+
+![Screenshot of the new Remember Me global setting](./RememberMe.jpg)
+
+Additionaly the code behind the login form has been updated to remove several annoying bugs. Unlike before, access for all remembered users will continue to work even when other users log in and out. Also, the What's New popup will now appear for all users on their 1st login of a new app version.
+
+### 🎥 On Screen Display (OSD)
+
+![Screenshot of the new On Screen Display (OSD)](./OSD.jpg)
+
+The new OSD provides quick access to the new chapter navigation functions, and is also the new way to access the playback info and subtitle section popups. This is only the start for the OSD. We have a lot of new and exciting functions we will be adding in the future!
+
+To access the menu simply press either UP, DOWN, OK, or PAUSE while in the video player. Like we said, the menus you previously accessed by pressing UP or DOWN are now available from the OSD.
+
+**Note:** The video position bar is not currently selectable from the OSD. It is informational only for now.
+
+If you want to step through the video position, simply press LEFT or RIGHT when the OSD is not on the screen, just like it's always been before. Rewind and Fast Forward also operate the same way. Simply press the button on the remote when the OSD is not shown.
+
+### 🎸 Music Video Support
+
+![Screenshot of Latest in Music Videos on the home view](./MusicVideos.jpg)
+
+Support for Music Videos has been added. If the user has access to the Music Videos library, it will appear in the My Media section and the Latest videos will show on the home view.
+
+### 🏎️ Quickplay Support Expanded To Entire App
+
+Stop wasting time and get your media playing ASAP! Quickplay support is now across the entire app. Simply highlight an item and  press play on your remote. TV Season? Check. Playlist? Check. Your Music library for a week long rager? Ya' better believe check.
```

Comment:
```suggestion
Stop wasting time and get your media playing ASAP! Quickplay support is now across the entire app. Simply highlight an item and press play on your remote. TV Season? Check. Playlist? Check. Your entire Music library for a week long rager? Ya' better believe check.
```

Might want to mention you can quick play an actor too. I think most people will assume this only works with "media" so it's not very intuitive that this will work on actors.

Reply:
We limited playback to 2000 items. I don't want people to get the idea it'll queue up 30,000 songs if that's what they have in their music library.

Reply:
That's fair. What if we add a NOTE somewhere that says quickplay playlists are limited to 2000 items max?

Reply:
Updated the copy to be.

Quickplay support is now across the entire app. Simply highlight an item and press play on your remote. The client will then queue up to 2000 items and start playing.